### DISCUSSION TITLE
NOTE mapping review

### DISCUSSION THREAD

**Discussion by Chesterguan** [chorus-ai/StandardsModule#84] [Results Sharing Questions] [answered]
Hi Guys:

I created the discussion and would love to get feedback and reviews from the teams about NOTE mapping. 
The purpose of the mapping is mainly to generate the fields for `note_type_concept_id ` and `note_class_concept_id ` for table `CDM.NOTE`. 

[NOTE mapping UF draft](https://uflorida-my.sharepoint.com/:x:/g/personal/ziyuan_guan_ufl_edu/EW2oIoAsQ6BHg-jVkmS3seMBo3GcQvAwj_fnrf5umigohw?e=RTiaR1)

Hopefully with the help of community, we could finalize the mapping and shared with all sites. 

@clermontg @del42  Please add more contents or ideas here if I missed anything.

**Comment by heschmidt04** [comment]
Folks, 

For those who have issues with Web Office 365 on Macintosh not allowing you to export to CSV (except to PDF format, in my case)  
I've attached the CSV file and the DDL for creating the table. 
I've loaded the CSV into our database and will use it to sleuth and review. 

Thanks!
Heidi 

```
[NOTE_mapping.csv](https://github.com/chorus-ai/StandardsModule/files/13923740/NOTE_mapping.csv)
```

**Reply by heschmidt04** [reply]
DDL didn't load in last post. Here it is. 

```
CREATE OR REPLACE TABLE notes_mappings_uflorida (
NoteType		VARCHAR(1000),
InpatientNoteType	VARCHAR(1000),
inpatient_note_type	VARCHAR(1000),
note_title		VARCHAR(1000),
note_source_value	VARCHAR(1000),
note_type_concept_id	NUMBER(38),
note_type_concept_code	VARCHAR(1000),
note_type_name		VARCHAR(1000),
note_class_concept_id	NUMBER(38),
note_class_concept_code	VARCHAR(1000),
note_class_name		VARCHAR(1000)
)
```

**Comment by p-talapova** [comment, marked as answer]
Hello everyone, 

I've conducted a semantic review of the mapping provided and documented some minor suggestions in the attached XLSX file. These findings include providing more specific categorizations where applicable, considering more generic terms for uncertain concepts, and addressing potential overlaps between outpatient and inpatient categorizations. These suggestions are intended as considerations rather than final decisions. That is why we appreciate your feedback!

[NOTE_mapping_suggestions.xlsx](https://github.com/chorus-ai/StandardsModule/files/14010812/NOTE_mapping_suggestions.xlsx)

**Reply by Chesterguan** [reply]
Thanks Polina, it's very helpful!

**Comment by Chesterguan** [comment]
Feels like the concept_ids selected here are not corresponding with the [broad SOPs](https://chorus-ai.github.io/data_acq_SOP/docs/Data-Uploading/Broad%20SOP)

@clermontg what should we do ?

**Reply by clermontg** [reply]
I agree with the general principle that one should map as finely as possible. The Broad SOP provided suggestions of mappings for common note classes as provided below. There were derived from ATHENA's LOINC codes for the NOTE domain using the LOINC vocab. UF codes are derived differently. For Discharge Summary, the corresponding entries are 
706531 and LP173221-5, from Domain Measurement Value and Class (Doc Type of Service). I am not sure what Polina said, but ultimately we need preferred maps from standards for the concepts below, and a preferred way to derived those maps. @p-talapova, can we engage on this? Alternatively, we resolve after submission while this is figured out.


**Broad SOP suggestions:**

Notes:
- History and Physical/admission (3030023; 34117-2)

- Progress notes (3000735; 11506-3)

- Medical Consults* (3020785;11488-4)

- OR, procedure reports (3018897;28570-0)

- Discharge summaries (3020091; 18842-5)

Reports:
- Radiology* (40771183; 68604-8 – 45879817; LA16679-5)

- Cardiac echo** (3018897; 28570-0)

- EEG** (3018897; 28570-0)

- Surgical pathology/Cytology (3031451; 34819-3)

**Comment by clermontg** [comment]
Cheater,

I will look at that discussion.

Gilles
